android/src/main/java/com/oney/WebRTCModule/ReactBridgeUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.oney.WebRTCModule;
+
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.ReadableType;
+
+public class ReactBridgeUtil {
+
+    public static String getMapStrValue(ReadableMap map, String key) {
+        ReadableType type = map.getType(key);
+        switch (type) {
+            case Boolean:
+                return String.valueOf(map.getBoolean(key));
+            //case Number:
+            //    return String.valueOf(map.getInt(key));
```

Comment:
I don't understand why you've left this comment(ed source code). Could you please elaborate?